### PR TITLE
Tbolt/3158 Update key state personnel page to reflect updated designs

### LINKED ISSUE #3158: [Feature] Update Key State Personnel page to show if no information has been entered
As a state reviewer, I would like to see where no response was provided for Medicaid office, so that I'm not confused that it might be an error.

**Description and related issues -**
We've determined that this section may be hard for the reviewer to understand whether a state has entered information, or if something is broken in the export view. We'd like to add in an indication that no information has been entered so the reviewer knows a field was left blank.

**What is the goal or purpose of this ticket?**
Make the export view easier for users to understand.

**Mockups or link to Figma**
When the Medicaid Director's name isn't entered, it is not showing on the export view, nor is there an indication that there should be a name if no information is entered. The phone number and email fields appear, but are blank. I think it's safe to assume that the the reviewer could see that the fields aren't filled out if all the fields exist together. In this empty state it does appear slightly broken. 
Screenshot of current

Screenshot of requested design in export view

### Acceptance criteria

This should include at least:
- [x] If a Medicaid Director's name was entered, it should be displayed in the export view. 
- [x] If a Medicaid Director's name was not entered, the field label Name: should be displayed in the export view. 
- [x] If a medicaid office address was entered, it should be displayed in the export view.
- [x] If a medicaid office address was not entered, "No response was provided" should be displayed in the export view.

### This task is done when…

- [ ] the APD Overview page matches the mock ups
- [ ] all acceptance criteria are met

### PR DESCRIPTION
**Pull request by tbolt**
resolves #3158 

**This pull request changes...**

- Updates State Medicaid Director Name & Address fields to the proposed design

**Important note**

- We set a default state (based on their affiliation afaik) so if ONLY the state is selected it will still show "No response was provided" for the address field

**This pull request was tested in the follow ways…**

- New APD Created, verified that the export view displayed as expected based on proposed design
- State Medicaid Director Name updated, verified that displayed OK
- State Medicaid Office address added, verified that displayed OK

**Steps to manually verify this change...**

1. Login with any account that can create an APD
2. Verify export view is as proposed with no data entered
3. Verify that when data is added it is displayed as normal

### This pull request is ready to review when...

- [ ] Automated tests are updated (and all tests are passing)
- [ ] The change has been documented
- [ ] Associated OpenAPI documentation has been updated
- [ ] Changelog is updated as appropriate
- [ ] The experience passes a basic manual accessibility audit (keyboard nav, screenreader, text scaling) OR an exemption is documented

### This pull request can be merged when…

- [ ] Code has been reviewed by someone other than the original author
- [ ] QA has verified the accessibility and functionality related to the change
- [ ] Design has approved the experience
- [ ] Product has approved the experience

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,16 +1,18 @@
...
-Anticipated release: May 17, 2021
+Anticipated release: June XX, 2021
 
 #### 🚀 New features
-- Allow multiple states to be selected in onboarding process ([#2638])
 
+- Allow multiple states to be selected in onboarding process ([#2638])
 - Update APD paths to include APD id ([#2589])
 - Updated delete dialog to use thematically consistent modal dialogs instead of basic alerts ([#2926])
 - New affiliation available for federal users ([#2929])
```
Comment (thetif) [inline CHANGELOG.md:10]: can you update this changelog too?

File: web/src/containers/viewOnly/ApdStateProfile.js [hunk 2]
```diff
@@ -49,31 +49,38 @@ const ApdStateProfile = ({ stateProfile, keyPersonnel }) => {
...
+    // Since we provide a default State don't check if falsy
+    if(!address1 && !address2 && !city && !zip) {
+      return ( <span>No reponse was provided</span> ) ;
+    }
+
+    return (
+      <address>
+        {address1}
+        <br />
+        {address2 &&
```
Comment (thetif) [inline web/src/containers/viewOnly/ApdStateProfile.js:64]: Can you put a !! in front of the first address2? I see what you are doing here, but it's a little confusing at first glance. I don't want someone to mistake it for a typo.

https://dev.to/sanchithasr/what-is-the-double-bang-operator-in-javascript-4i3h

**Review by thetif — APPROVED** [review]

**Comment by thetif** [comment]
content matches mock ups

**Review by beparticular — APPROVED** [review]

**Review by NAretakis — APPROVED** [review]
Good to go.

**Pull request merged**